RealmTasks React/components/login.js

```diff
@@ -0,0 +1,120 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+'use strict';
+
+import React from 'react';
+import { AsyncStorage, Text, TextInput, View, Button } from 'react-native';
+import RealmTasks from './realm-tasks';
+import styles from './styles';
+import TodoApp from './todo-app';
+import config from '../config';
+
+
+export default class LoginScreen extends React.Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {login: '', password: ''};
+        this.user = null;
+    }
+
+    login () {
+        RealmTasks.login(
+            this.state.login,
+            this.state.password,
+            (error, realm) => {
+                if (error) {
+                    this.state.error = error.message;
```

Comment:
I believe you should use `this.setState({ error: error.message })` instead of setting `this.state` directly. Then you won't need to call `this.forceUpdate()`.